vault/datadog_checks/vault/metrics.py

```diff
@@ -207,10 +207,10 @@
     'vault_postgres_get': 'vault.postgres.get',
     'vault_postgres_delete': 'vault.postgres.delete',
     'vault_postgres_list': 'vault.postgres.list',
-    'vault_raft_put': 'vault.raft.put',
-    'vault_raft_get': 'vault.raft.get',
-    'vault_raft_delete': 'vault.raft.delete',
-    'vault_raft_list': 'vault.raft.list',
+    'vault_raft_storage_put': 'vault.raft.put',
+    'vault_raft_storage_get': 'vault.raft.get',
+    'vault_raft_storage_delete': 'vault.raft.delete',
+    'vault_raft_storage_list': 'vault.raft.list',
```

Comment:
The telemetry docs also have `vault_raft_get/put/delete/list` metrics as well. Instead of replacing the previous metrics, can we add the `vault_raft_storage_...` metrics to the list?
https://www.vaultproject.io/docs/internals/telemetry#integrated-raft-storage-health

Reply:
Indeed, both exists and does not have the same semantic: `vault_raft_storage_*`are timers and `vault_raft_*` are summaries. 
I do not completely understand the difference between the two, `raft_storage` seems to be related to FSM but I don't what that stands for and how it relates to raft backend storage.

`vault_raft_get/put/delete/list` meaning seems closer to the other storage backend metrics, `vault_postgres_*` for instance.

Reply:
> `vault_raft_storage_*`are timers 
That's interesting, I don't believe I've seen other vault metrics of that type. Would you be able to share a snippet of what the prometheus output looks like for that metric?

>vault_raft_get/put/delete/list meaning seems closer to the other storage backend metrics, vault_postgres_* for instance.

I understand what you mean by keeping the naming of storage backend metrics consistent. However, for vault, we have tried to keep the metric names consistent to the prometheus names largely due to the sheer number of metrics supported and to prevent incompatibility as new metrics are introduced.

Keeping the metric `vault.raft_storage.get` makes it very obvious which metric we're actually referring to. What do you think?

Reply:
My priorities have shifted away from vault recently.
I hope I'll have some time in the next week to provide you some of my prometheus metrics.